### ISSUE TITLE
Empty string always passed as first argument to mixin

### ISSUE THREAD

**Issue by billyjanitsch** [postcss/postcss-mixins#56]
```js
const myMixin = (rule, x, y) => {
  console.log(x, y)
  return {}
}
```

```css
.example {
  @mixin myMixin; /* => '' undefined */
  @mixin myMixin 50px; /* => '50px' undefined */
}
```

I would expect the first invocation to log `undefined undefined`, but instead it receives an empty string as the first argument. I assume this is a bug, since the second invocation logs `50px undefined` instead of `50px ''`.

I don't think passing an empty string is good, because I might want to check whether the mixin received an argument or not.

**Comment by ai** [comment]
Strange O_o. I will try to look on this week (today I am already drunk :D). Feel free to ping me again if I will not fix it before Friday.

**Comment by ai** [comment]
Fixed c2ded4b

Thanks for report

**Comment by ai** [comment]
Released in 4.0.2
